lib/stack_master/utils.rb

```diff
@@ -4,7 +4,7 @@ module Utils
 
     def hash_to_aws_parameters(params)
       params.inject([]) do |params, (key, value)|
-        params << { parameter_key: key, parameter_value: value }
+        params << { parameter_key: key, parameter_value: value.to_s }
```

Comment:
Thanks. Will look into that. I guess we need some extra test coverage around that case.